wmn-data.json

```diff
@@ -3098,6 +3110,18 @@
         "m_code" : 404,
         "known" : ["mohammed01", "mohammed03"],
         "cat" : "gaming",
+        "valid" : true
+       },        
+       {
+        "name" : "likeevideo",
+        "uri_check" : "https://likee.video/@{account}",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "(@",

```

Comment:
String is too simple and likely to cause false positives. Suggest `https://img.like.video/`. It looks like the missing profile pages use the HTTP and the pages with profiles use HTTPS for this URL. Please check.

Reply:
@WebBreacher : `Done!` re change of `e_string` for `likeevideo`